rosapi/interface.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+
+class Interface():
+  def __init__(self, api, interface):
+    self.interface_name = interface
+    self.api = api
+
+  def checkinterface(self):
```

Comment:
I disagree, this method only checks if the interface exists. It doesn't make sense to name it "print. "